collector/index.mdx

```diff
@@ -29,3 +29,33 @@ The collector makes only outbound connections, it does not listen on any ports.
 Heroku-based setup, where the collector listens on the dyno's PORT for Heroku to consider the application running,
 as well as to receive database logs by serving as a Heroku log drain.
 
+## Data collection and Snapshot submission Frequency
+
+The pganalyze collector gathers metrics and metadata through various snapshots, each with specific cadences.
```

Comment:
```suggestion
The pganalyze collector gathers metrics and metadata through various "snapshots", each with specific cadences.
```
I think because we use the term "snapshot" in a very specific way when talking about the collector, we should put the first occurrence in quotes to indicate that this term has special meaning here.